html/index.html

```diff
@@ -154,7 +170,8 @@ <h2 class="my-0 fw-semibold">Nautobot Core</h2>
                             <li class="list-group-item"><a href="https://nautobot-ansible.readthedocs.io/en/latest/#"
                                     class="link-dark">Nautobot Ansible Collection</a><span
                                     class="badge text-bg-dark rounded-pill ms-1">Ansible</span></li>
-                            <li class="list-group-item"><a href="https://docs.nautobot.com/projects/pynautobot/en/latest/index.html"
+                            <li class="list-group-item"><a
+                                    href="https://docs.nautobot.com/projects/pynautobot/en/latest/index.html"
```

Comment:
Just checking - should this and others be changed to `/stable/` or is `/latest/` correct for these projects?

Reply:
Most other projects serve the latest docs from develop by default. Thinking back, it might've been because it was the only way to get the updated docs online quickly. Now that things have settled down, we could switch the default to stable for most of them.

Then again, apps and such don't see a lot of docs changes on a regular basis.